### PR TITLE
Allow for bad replicas to be scaled down

### PR DESCRIPTION
**Pull request by jkatz**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change, as applicable?
 - [x] Have you tested your changes on all related environments with successful results, as applicable?

**Type of Changes:**

 
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the new behavior (if this is a feature change)?**

Previously, the scaledown command would only list out healthy
replicas, not replicas that were in a bad state. However, these
may be the exact replicas that one would want to scale down,
given they are unhealthy.

Issue: [ch9253]

### CODE REVIEW

File: internal/util/failover.go [hunk 1]
```diff
@@ -137,20 +137,34 @@ func GetPod(clientset kubernetes.Interface, deploymentName, namespace string) (*
...
+
+	// if we are not including the primary, determine if we are including busted
+	// replicas or not
 	if !includePrimary {
-		roleSelector = fmt.Sprintf("%s=%s", config.LABEL_PGHA_ROLE, config.LABEL_PGHA_ROLE_REPLICA)
-	} else {
-		roleSelector = config.LABEL_PG_DATABASE
+		if includeBusted {
+			// include all Pods that identify as a database, but **not** a primary
+			// pg-cluster=clusterName,pgo-pg-database,role!=config.LABEL_PGHA_ROLE_PRIMARY
```
Comment (jmckulk) [inline internal/util/failover.go:160]: I think having something like `selector + role!=config.LABEL_PGHA_ROLE_PRIMARY` would make it more clear what this line is doing

File: internal/util/failover.go [hunk 2]
```diff
@@ -137,20 +137,34 @@ func GetPod(clientset kubernetes.Interface, deploymentName, namespace string) (*
...
+			selector += fmt.Sprintf(",%s!=%s", config.LABEL_PGHA_ROLE, config.LABEL_PGHA_ROLE_PRIMARY)
+		} else {
+			// include all Pods that identify as a database and have a replica label
+			// pg-cluster=clusterName,pgo-pg-database,role=replica
```
Comment (jmckulk) [inline internal/util/failover.go:164]: same here `selector + role=replica`

**Review by jmckulk — APPROVED** [review]
Have a couple comments about the comments but the changes make sense to me

File: internal/util/failover.go (hunk 1 above)
Comment (jkatz) [reply internal/util/failover.go:160]: This shows the full selector string though.

**Pull request merged**